themes/SuiteP/css/suitep-base/navbar.scss

```diff
@@ -1064,7 +1064,7 @@
 }
 
 .desktop_notifications > .dropdown-menu {
-  left: auto;
+  left: -82%;
```

Comment:
This also works for mobile? I dislike having these percentages as large as that, it's almost masking the real problem I feel. If I enlarge my text on my browers (cause I often take my glasses off) will it still be in a reasonable position?

Reply:
@samus-aran - you are correct, we shouldn't use it like that.
Do you have a suggestion on how to fix it correctly? Do you prefer I will remove it from the PR for now (I'm not sure when I will have the time to find the best solution)?